src/hooks/spacewalk/useIssuePallet.tsx

```diff
@@ -46,7 +47,10 @@ export function useIssuePallet() {
           return undefined;
         }
 
-        return api.tx.issue?.requestIssue(amount, vaultId);
+        const integerAmount = parseFloat(amount).toFixed(0);
```

Comment:
Why is the `parseFloat` conversion necessary?

Reply:
@ebma You're right it was not clear. ✅  I changed it to be handled by `Big`. I also noticed that the user could mess up `api.tx.issue?.requestIssue(compactAmount, vaultId)` in many ways, because `compactAmount` was not properly checked to be a valid `u128` type. I added the `isU128()` function along with the tests. Now it works perfectly and safely.